### PR TITLE
Novaxr rename

### PR DESCRIPTION
**Pull request by Andrey1994**
Changes in this PR:

* rename novaxr
* add string output from config board method
* drop tty ports for MacOS

### CODE REVIEW

File: OpenBCI_GUI/Board.pde [hunk 1]
```diff
@@ -102,7 +103,7 @@ abstract class Board implements DataSource {
 
     public abstract boolean isConnected();
 
-    public abstract boolean sendCommand(String command);
+    public abstract Pair <Boolean, String> sendCommand(String command);
```
Comment (retiutut) [inline OpenBCI_GUI/Board.pde:106]: Nice! This makes sense as a `Pair`.

File: OpenBCI_GUI/BoardAuraXR.pde [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import brainflow.*;
 
 import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.tuple.Pair;
```
Comment (retiutut) [inline OpenBCI_GUI/BoardAuraXR.pde:4]: 👍

File: OpenBCI_GUI/BoardBrainFlowStreaming.pde [hunk 3]
```diff
@@ -1,5 +1,7 @@
 import brainflow.*;
 
+import org.apache.commons.lang3.tuple.Pair;
```
Comment (retiutut) [inline OpenBCI_GUI/BoardBrainFlowStreaming.pde:3]: Is this import being duplicated in other .pde files for unit testing purposes? @Andrey1994

**Review by retiutut — APPROVED** [review]

File: OpenBCI_GUI/ControlPanel.pde [hunk 4]
```diff
@@ -1272,6 +1272,10 @@ class ComPortBox {
         for (SerialPort comPort : comPorts) {
             for (String name : names) {
                 if (comPort.toString().startsWith(name)) {
+                    // on macos need to drop tty ports
+                    if (isMac() && comPort.getSystemPortName().startsWith("tty")) {
+                        continue;
```
Comment (retiutut) [inline OpenBCI_GUI/ControlPanel.pde:1277]: 👍

File: OpenBCI_GUI/BoardBrainFlowStreaming.pde (hunk 3 above)
Comment (Andrey1994) [reply OpenBCI_GUI/BoardBrainFlowStreaming.pde:3]: idk about processing but in java you need to import everywhere

**Pull request merged**